app/generate/page.tsx

```diff
@@ -1,11 +1,25 @@
+'use client';
```

Comment:
Why do you need this here ? 😊

Reply:
Because in Next 13, in the app folder the pages are server component by default. 

If i don't have it i have this error that appear : 
<img width="1003" alt="Screenshot 2023-02-27 at 14 23 52" src="https://user-images.githubusercontent.com/72607059/221575306-76eadd6c-1ac5-43ea-adaa-b640da1bfb78.png">